cpp/examples/ide_secir.cpp

```diff
@@ -98,6 +98,6 @@ int main()
     sim.advance(tmax);
 
     sim.get_result().print_table({"S", "E", "C", "I", "H", "U", "R", "D "}, 16, 8);
```

Comment:
```suggestion
       auto interpolated_results = mio::interpolate_simulation_result(sim.get_result());
       interpolated_results.print_table({"S", "E", "C", "I", "H", "U", "R", "D "}, 16, 8);
```
dt is 0.01 so you are printing a lot of (maybe uninteresting) results.

Reply:
This is actually a nice tool!